packages/components/src/slot-fill/bubbles-virtually/slot-fill-context.ts

```diff
@@ -8,8 +8,12 @@ import { proxyMap } from 'valtio/utils';
  */
 import { createContext } from '@wordpress/element';
 import warning from '@wordpress/warning';
+/**
+ * Internal dependencies
+ */
+import type { SlotFillBubblesVirtuallyContext } from '../types';
 
-const SlotFillContext = createContext( {
+const initialContextValue: SlotFillBubblesVirtuallyContext = {
```

Comment:
After typing the file and renaming it to `.ts`, did you also mean to remove the `@ts-nocheck` comment?

Reply:
I thought I turned it off, but it leaked. :sweat_smile: